projects/core/src/lib/plugins/change-event-plugin.ts

```diff
@@ -0,0 +1,33 @@
+import type {MaskitoPlugin} from '../types';
+
+export function maskitoChangeEventPlugin(): MaskitoPlugin {
+    return element => {
+        if (element.isContentEditable) {
+            return;
+        }
+
+        let value = element.value;
+
+        const focusListener = (): void => {
```

Comment:
Let's call it `valueListener`  and use the same on both cases, since there's no difference?

Reply:
Fixed in https://github.com/taiga-family/maskito/pull/1338/commits/55192a78648be4d4c17068c53347d8ca5af5115a